src/bgw_policy/chunk_stats.c

```diff
@@ -154,17 +157,19 @@ static ScanTupleResult
 bgw_policy_chunk_stats_update_tuple_found(TupleInfo *ti, void *const data)
 {
 	TimestampTz *updated_last_time_job_run = data;
-	HeapTuple tuple = heap_copytuple(ti->tuple);
-	BgwPolicyChunkStats *chunk_stats = STRUCT_FROM_TUPLE(ti->tuple,
-														 ti->mctx,
-														 BgwPolicyChunkStats,
-														 FormData_bgw_policy_chunk_stats);
+	bool should_free;
+	HeapTuple tuple = ExecFetchSlotHeapTuple(ti->slot, false, &should_free);
+	HeapTuple new_tuple = heap_copytuple(tuple);
```

Comment:
I am puzzled by this change: What do we gain by introducing a call to ExecFetchSlotHeapTuple at line 161.? We make a copy of the tuple here.